sbt-scalafmt/build.sbt

```diff
@@ -6,9 +6,11 @@ buildInfoPackage := "com.lucidchart.sbt.scalafmt"
 
 libraryDependencies ++= Seq(
   "com.google.code.findbugs" % "jsr305" % "3.0.2" % Provided, // fixes warnings about javax annotations
-  "com.google.guava" % "guava" % "19.0"
+  "com.google.guava" % "guava" % "19.0",
+  "com.michaelpollmeier" %% "colordiff" % "0.8"
 )
 
+resolvers += Resolver.mavenLocal
```

Comment:
I was doing 1 more pass of the code before merging, and I am wondering why you added this line? Any particular reason, or could that be removed from the PR? I'm not opposed to keeping it if there is a good reason. (SBT configs and build files are new territory for me)

Reply:
that wasn't supposed to get committed, I'll remove it right now.